webhook.rst

```diff
@@ -19,17 +19,26 @@ Installation
 Usage in Combination with the Mailer Component
 ----------------------------------------------
 
-When using a third-party mailer, you can use the Webhook component to receive
-webhook calls from the third-party mailer.
+When using a third-party mailer provider, you can use the Webhook component to
```

Comment:
We are using "provider" in the notifier and mailer docs, let's use the same here.